v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/templates/ProtegrityDataTokenization.java

```diff
@@ -354,4 +345,31 @@ public static PipelineResult run(ProtegrityDataTokenizationOptions options,
 
     return pipeline.run();
   }
+
+  private static PCollection<Row> convertJsonToRow(ProtegrityDataTokenizationOptions options,
```

Comment:
looks like PTransform, can we try to use it? or for some reason, it not possible here?